packages/react/src/page-alert/PageAlertTitle.tsx

```diff
@@ -4,21 +4,24 @@ import { Text } from '../text';
 
 export type PageAlertTitleProps = PropsWithChildren<{
 	as?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+	/** @deprecated use `hasCloseButton` instead */
```

Comment:
Can you `console.warn()` this too?

Reply:
I added console.warn() to the function I was using previously for throwing the error when both are used. now it only throws warnings.